src/main/groovy/org/pih/warehouse/monitoring/SentryServletContainerInitializer.groovy

```diff
@@ -0,0 +1,42 @@
+package org.pih.warehouse.monitoring
+
+import io.sentry.Sentry
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+
+import javax.servlet.ServletContainerInitializer
+import javax.servlet.ServletContext
+import javax.servlet.ServletException
+
+/**
+ * Configure the sentry-servlet library to send URL/user-agent data to Sentry.
+ *
+ * https://docs.sentry.io/platforms/java/guides/servlet/#configure
```

Comment:
Can you elaborate on what this is for / why we need it? As a comment to the code if possible.

Reply:
I added an explanatory comment as follows in 23788a183:

```
/**
 * Configure Sentry at application startup.
 *
 * Sentry integration occurs in three places. This class, which Sentry
 * finds via a file named javax.servlet.ServletContainerInitializer, sets
 * global parameters, including information about the release. The init
 * call also (magically) makes it possible for Sentry to retrieve url
 * and user-agent information from the application server.
 *
 * See also SentryInterceptor, which provides Sentry with additional
 * data at each page load, and logback.xml, for logging integration.
 *
 * https://docs.sentry.io/platforms/java/guides/servlet/#configure
 */
```